### PR TITLE
Drop D1 support and migrate to swarm v6

### PR DESCRIPTION
**Pull request by tiyash-basu-sociomantic**
This PR contains breaking changes and should be targeted on a `15.x.x` branch.

### CODE REVIEW

**Comment by tiyash-basu-sociomantic** [comment]
Updated to use swarm `v6.0.0-rc.3`.

File: src/dhtproto/node/request/Listen.d [hunk 1]
```diff
@@ -59,7 +59,7 @@ public abstract scope class Listen : SingleChannel
 
         Keeps sending new records in dht channel to client until channel
         gets removed or client closes connection
-    
+
```
Comment (matthias-wende-sociomantic) [inline src/dhtproto/node/request/Listen.d]: seems unrelated

File: src/dhtproto/node/request/Listen.d [hunk 2]
```diff
@@ -108,7 +108,7 @@ public abstract scope class Listen : SingleChannel
 
             if (disconnect_detector.disconnected)
                 return;
-        
+
```
Comment (matthias-wende-sociomantic) [inline src/dhtproto/node/request/Listen.d]: ditto

File: src/dhtproto/node/request/Put.d [hunk 3]
```diff
@@ -88,25 +88,16 @@ public abstract scope class Put : SingleKey
...
 
-    protected size_t recordSizeLimit ( )
-    {
-        return RecordBatcher.DefaultMaxBatchSize;
-    }
+    protected size_t recordSizeLimit ( );
 
     /***************************************************************************
-    
+
```
Comment (matthias-wende-sociomantic) [inline src/dhtproto/node/request/Put.d]: spaces :)

File: src/dhtproto/node/request/Put.d [hunk 4]
```diff
@@ -132,7 +123,7 @@ public abstract scope class Put : SingleKey
             this.writer.write(DhtConst.Status.E.EmptyValue);
             return;
         }
-        
+
```
Comment (matthias-wende-sociomantic) [inline src/dhtproto/node/request/Put.d]: ditto

File: src/fakedht/request/Put.d [hunk 5]
```diff
@@ -58,7 +59,7 @@ public scope class Put : Protocol.Put
     {
         return true;
     }
- 
+
```
Comment (matthias-wende-sociomantic) [inline src/fakedht/request/Put.d]: ditto

**Review by matthias-wende-sociomantic — COMMENTED** [review]
lots of spaces :D

**Comment by tiyash-basu-sociomantic** [comment]
Removed all whitespace changes. Thanks for pointing out!
